src/components/ProductCard/ProductCard.tsx

```diff
@@ -1,36 +1,41 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
-import './cardphone.scss';
+import './productcard.scss';
+import { Phone } from '../../types/Phone';
 
-export const CardPhone: React.FC = () => {
+type Props = {
+  phone: Phone;
+};
+
+export const ProductCard: React.FC<Props> = ({ phone }) => {
   return (
-    <article className="card">
+    <article className="card" key={phone.id}>
       <img
-        src="../img/phones/apple-iphone-14-pro/spaceblack/00.webp"
+        src={`../${phone.images[0]}`}
         alt="Iphone 14 Midnight"
         className="card__image"
       />
 
       <h2 className="card__title">
-        Apple iPhone 14 Pro 128GB Deep Purple (MQ0G3)
+        {phone.name} (MQ023)
```

Comment:
```suggestion
        {phone.name}
```

leave only `phone.name`, cuz we don't have info about models for now